### ISSUE TITLE
[Quest Item][WoTLK][Zul'Drak] Unliving Choker

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4641]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
So the choker dropped when i was in party with a friend. I looted the item he was not in range to loot the item. I clicked it to get the quest and ever since then the choker has not dropped for him.
We have left group. Still no drop for him. He restarted the game still no drop for him. He deleted the cache on a 2nd resart and currently there is still no drop.
**Expected behaviour**: Tell us what should happen instead
The item should drop if the player is not on the quest. The item should only bee seen if the player has not started the quest.
**Steps to reproduce the problem**:

1. Be in party pick up quest item start quest.  With out other members looting item.
2. It wont drop for other party member off another mob.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

This item should drop as long the the player has not started the quest.
http://www.wowhead.com/item=38660/unliving-choker

The quest giving from item.
http://www.wowhead.com/quest=12631/an-invitation-of-sorts

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TheRealestMcCoy** [comment]
Just a side note I am hearing you must complete cleansing draktharon. This is not the case it does not have any pre requirements for it to drop. I have not done it and did receive the choker.

**Comment by TeridanDW** [comment]
Hello, thank you for the report @TheRealestMcCoy .
Regarding the issue:
 - in the DB we have:
``` sql
item = 38660 , drop chance = 100% , QuestRequired = 0
item = 38673 , drop chance = 100% , QuestRequired = 0
```
 - both items http://wotlk.openwow.com/item=38673 and http://wotlk.openwow.com/item=38660 , even though have diffrent quests leads to the same end. For a player it's not posible to get both of them.
The condition "should" be to be level 74+ and haven't done quest http://wotlk.openwow.com/quest=12238 and you kill a troll = 100% drop of item http://wotlk.openwow.com/item=38660
Screenshot from PTR from first kill:
![image](https://cloud.githubusercontent.com/assets/6797611/24446190/471ca214-1475-11e7-88eb-f1c84efa12f6.png)

 And if the player has completed the quest http://wotlk.openwow.com/quest=12238
![image](https://cloud.githubusercontent.com/assets/6797611/24446583/b1bbbc58-1476-11e7-9027-47eddebe06e9.png)
![image](https://cloud.githubusercontent.com/assets/6797611/24446633/e3f5266e-1476-11e7-9e1c-4b21a264fa24.png)

Issue related to #365 and #1554

I couldn't find any issue here, if you still something is wrong please comment below, closing for now.

**Comment by TheRealestMcCoy** [comment]
I dont know what the issue was. BUt we killed 100s of mobs trying to get unliving to drop. It would not drop. I never did any of the troll quest line in Grizzly hills and it dropped for me.

It may be because he was on the cleansing quest but had not completed it yet cause he had not cleanse drak locking him from the unliving. 

Yes if you complete the quest cleansing drak you should get the Writhing Choker.

Being he had not completed the quest the unliving should have dropped for him.

**Comment by TheRealestMcCoy** [comment]
To test try being in a party.
Have one player on cleansing drak.
Go kill mobs where you both are not in loot range of each other.
See who get the choker first. (Go ahead and start the quest.)
Now start farming mobs together see if it drops.

**Comment by TeridanDW** [comment]
Tried, same result, 100% drop chance for both players . Maybe it's just working properly on PTR.

**Comment by TheRealestMcCoy** [comment]
Mat be.

**Comment by Rushor** [comment]
some conditions for the drop are missing. currently it's set to drop if queststatus of 12238 = none, but it should also drop if quest 12238 is taken or completed (not rewarded)

**Comment by Rushor** [comment]
DB/Item: Unliving Choker - Show loot if quest Cleansing Drak\'Tharon has been taken/completed/not taken

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4641
* commit: 79535cfb9e323e9f91dbc890d604ff115505b67a

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TheRealestMcCoy** [comment]
So does this mean unliving choker will drop now regardless of cleansing drak?

**Comment by Rushor** [comment]
it will drop if you don't have quest http://wotlk.openwow.com/quest=12238 or if you have quest http://wotlk.openwow.com/quest=12238 taken or you have quest http://wotlk.openwow.com/quest=12238 completed (not rewarded)

**Comment by TheRealestMcCoy** [comment]
You guys are awesome I am just glad I can do a small part by reporting things that can be improved.

**Comment by roblockster** [comment]
I have Cleansing Drak'Tharon taken (not turned in) and item won't drop.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
